packages/fxa-settings/src/pages/ResetPassword/index.test.tsx

```diff
@@ -139,6 +152,8 @@ describe('PageResetPassword', () => {
       fireEvent.click(await screen.findByText('Begin reset'));
     });
 
+    expect(GleanMetrics.resetPassword.submit).toHaveBeenCalledTimes(1);
```

Comment:
Should we also add a check for a `submit` ping when account is unknown or requests are throttled?

Reply:
👍 couldn't hurt.